prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts

```diff
@@ -170,6 +170,21 @@ export class AdjudicationResource {
       );
   }
 
+  public logEmailInitiated(enrolleeId: number): NoContent {
+    return this.apiResource.post<NoContent>(`enrollees/${enrolleeId}/email-initiated`)
+      .pipe(
+        map(() => {
+          this.toastService.openErrorToast('Enrollee Email Initiated has been sent');
+        }),
+        catchError((error: any) => {
+          this.toastService.openErrorToast('Enrollee Email Initiated could not be logged');
```

Comment:
Success and error toast reads a bit funny.  Could it be `"Enrollee email event could not be created"`, or something similar?